settings.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dictionaries": {
+    "build-in": ["en_US", "en_GB"],
+    "custom": ["assets/dic/extra-words.dic"]
```

Comment:
Great point, @stoe. Addressed in [4ea1a47](https://github.com/github/training-kit/pull/446/commits/4ea1a473d6fdd867efaa012319dfb2f0c47f79db).